imgui.cpp

```diff
@@ -1532,12 +1532,6 @@ FILE* ImFileOpen(const char* filename, const char* mode)
     ImTextStrFromUtf8(&buf[0], filename_wsize, filename, NULL);
     ImTextStrFromUtf8(&buf[filename_wsize], mode_wsize, mode, NULL);
     return _wfopen((wchar_t*)&buf[0], (wchar_t*)&buf[filename_wsize]);
-#elif defined(__NN_AARCH64__)
-	std::string mount(HOST_MOUNT_NAME ":/"); // keep and load files from the developer host
-	std::string path(filename);
-	std::string fullpath = mount + path;
-
-	return fopen(fullpath.c_str(), mode);
 #else
```

Comment:
Is this used anywhere else? Did you make sure this change doesn't break some other code that assumes the host file-system will be used?

Reply:
Should be fine, i'm pretty sure @bbi-markslemko added this so that the file would save to the host but since we now want it on the device, it should be ok to change it

Reply:
Yeah this function is only used internally by imgui to save logs/ini file, both those cases are handled by the main PR. Saving to host wasn't actually working on switch, but now it does!